vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -98,6 +98,9 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
+      if new_result[:ems_ref].nil? # Typically this happens for kubernetes services
+        new_result[:ems_ref] = "#{new_result[:namespace]}_#{new_result[:name]}"
```

Comment:
At the moment the only entity that is doing this is services and since it's special I don't want to blindly extend this to all other entities. For example for pods would be a little problematic because two different pods could have the same name in two different moments in time and the only way to recognize that they're different is to use the uid (which is the ems_ref).